corehq/apps/cloudcare/views.py

```diff
@@ -385,7 +385,9 @@ def _format_user(self, user_json):
             'last_name': user.last_name,
             'phoneNumbers': user.phone_numbers,
             'user_id': user.user_id,
-            'location': user.sql_location.to_json() if user.sql_location else None,
+            'location': user.sql_location.to_json()
```

Comment:
I was not sure if there is an equivalent on the web user with a different name. I noticed a `location` and `location_id` attribute but was not sure if they contain the same kind of information.

Reply:
you can use `user.get_sql_location(self.domain)` for both types of users